playbooks/ec2-metadata-instance.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: Drop packets to the Amazon EC2 metadata instance
+  hosts: streisand-host
+  gather_facts: no
+
+  remote_user: "root"
+  become: true
+
+  tasks:
+    - raw: route add -host 169.254.169.254 reject
```

Comment:
> @nopdotcom  I think this needs to be a persistent firewall rule, at least after provisioning finishes.

@alimakki I believe @nopdotcom is correct about this. I don't think this will persist across reboots. I think you can achieve persistence ifupd or systemd.